Orientation Maps/DPo/AverageHashHammingDistances.csv

```diff
@@ -0,0 +1,321 @@
+fingerprints,63,189,77,162,176,88,228,214,200,201,215,229,177,89,163,188,76,62,74,60,48,149,175,161,203,217,216,202,160,174,148,49,61,75,59,71,65,170,164,158,206,212,213,207,159,165,171,64,70,58,8,198,66,72,167,99,173,211,205,239,238,204,210,172,166,98,73,199,67,9,14,28,129,101,115,303,317,288,277,263,262,276,289,316,302,114,100,128,29,15,17,116,102,314,300,248,260,274,275,261,249,301,315,103,117,16,12,113,107,311,305,265,271,259,258,270,264,304,310,106,112,13,39,11,104,110,138,306,312,299,272,266,267,273,298,313,307,139,111,105,10,38,21,35,108,120,134,295,281,256,242,243,257,280,294,135,121,109,34,20,36,22,137,123,309,282,296,269,241,255,254,240,268,297,283,320,308,122,136,23,37,33,27,132,126,318,287,293,244,250,278,279,251,245,292,286,319,127,133,26,32,18,24,30,125,131,119,290,284,253,247,246,252,285,291,118,130,124,31,25,19,42,4,56,180,194,81,95,143,157,209,235,221,220,234,208,156,142,94,80,195,181,5,57,43,55,7,41,69,197,183,96,168,82,154,140,222,236,237,223,141,155,83,97,169,182,68,196,40,54,6,192,78,186,2,50,44,151,145,93,87,179,227,233,232,226,86,178,92,144,150,45,3,51,79,187,193,185,191,47,53,1,146,152,84,90,230,224,218,219,225,231,91,85,153,147,52,46,190,184

```

Comment:
what do you mean by average Hamming distance?

Reply:
Average hash is one of the hash techniques I am working on and average hash hamming distance file has hamming distances got using average hash